fr/policy/terms.md

```diff
@@ -1,51 +1,51 @@
 
-Nous nous réservons le droit de supprimer du contenu présumé contrefait, sans préavis, à notre seule discrétion, et sans responsabilité envers vous. Dans des circonstances appropriées, Retrospring sera également résilier le compte d'un utilisateur si l'utilisateur est déterminé à être un récidiviste.
+Nous nous réservons le droit, à notre entière discrétion, de supprimer du contenu présumé illicite, et ce, sans préavis ni responsabilité à votre égard. Retrospring peut, si les circonstances l'y obligent, résilier le compte d'un utilisateur en cas de violations répétées.
 
-*Entrée en vigueur: le 10 mai 2015*
+*Entrée en vigueur : le 10 mai 2015*
 
 <sup>
 Translated from en/policy/terms.md revision 7f6cd02e6efe2e78322ab9b651c32606f95695f4
 </sup>
 
 <sup>
-WARNING: IN NO CASE SHALL THE TRANSLATED VERSION BE CONSIDERED AS THE MOST RELIABLE SOURCE OF THE TERMS OF SERVICE/PRIVACY POLICY. In case an updated version of the document is available in the English version or the translation might be incorrect, the English version should always be considered as the most reliable source of the document.
+IMPORTANT : LA VERSION TRADUITE NE DOIT EN AUCUN CAS ÊTRE CONSIDÉRÉE COMME LA SOURCE LA PLUS FIABLE DES CONDITIONS GÉNÉRALES D'UTILISATION/DE LA POLITIQUE DE CONFIDENTIALITÉ. Étant donné qu'une nouvelle version de ce document peut être disponible dans sa version originale (anglaise) et que la traduction peut présenter des erreurs, la version originale doit toujours être considérée comme la source la plus fiable de ce document.
 </sup>
 
 <sup>
-ANY TRANSLATED NON-ENGLISH DOCUMENT IS CONSIDERED INVALID UNDER ANY CIRCUMSTANCE, AN ACCOMPANYING VALID ENGLISH WRITTEN DOCUMENT MUST BE PROVIDED AND CITED TO, IN NO CASE SHALL THE AUTHORS OR THE COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY RESULTING FROM ANY TRANSLATED NON-ENGLISH DOCUMENTS, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE RELATING SOFTWARE.
```

Comment:
Actually, it's supposed to stay english, it's a legal footnote made specifically so ***any translation*** is invalid and whoever is running the site cannot be held liable in the court of law, the fact it's english is insurance that we wrote it.